sssom/cli.py

```diff
@@ -89,6 +89,28 @@
     help="Fields.",
 )
 
+predicate_filter_option = click.option(
+    "-F",
+    "--mapping-predicate-filter",
+    multiple=True,
+    help="A list of predicates or a file path containing the list of predicates to be considered.",
+)
+
+
+def _get_list_of_predicates(predicate_filter: tuple) -> list:
+    
+    if all(":" in pred for pred in list(predicate_filter)):
+        mapping_predicates = list(predicate_filter)
+        # TODO: Cross-check if the predicate is a valid one 
+        # For e.g. check for typos etc.
+    elif os.path.isfile(predicate_filter[0]):
+        with open(predicate_filter[0], "r") as f:
+            mapping_predicates = f.read().splitlines()
+    else:
```

Comment:
Should be possible to be empty

Reply:
added an if statement below before calling the function.